install-validator/internal/kubernetes/client.go

```diff
@@ -0,0 +1,113 @@
+package kubernetes
```

Comment:
Lets not use "kubernetes" as package name.
Its leads to incorrect assumption while reading code that this is a kubernetes package.
You can call this pks as k8s_utils.